adapter_pipelines/format_map_example.json

```diff
@@ -1,12 +1,18 @@
 {
-  ".bam": "bam",
-  "_metrics": "metrics",
-  ".txt":"txt",
-  ".log":"log",
-  ".pdf":"pdf",
-  ".results":"results",
-  ".theta":"theta",
-  ".cnt":"cnt",
-  ".time":"time",
-  ".model":"model"
+  ".bam$": "bam",
+  "_metrics$": "metrics",
+  ".txt$":"txt",
+  ".log$":"log",
+  ".pdf$":"pdf",
+  ".results$":"results",
+  ".theta$":"theta",
+  ".cnt$":"cnt",
+  ".time$":"time",
+  ".model$":"model",
+  ".bai$": "bai",
+  ".tsv$": "tsv",
+  ".mtx$": "mtx",
+  ".h5$": "h5",
+  ".csv$": "csv",
+  ".zarr!": "zarr_format"
```

Comment:
Why would this check `.zarr!` and not `.zarray`?

Reply:
`.zarray` is just one of the "`zarr`" files family, in this case, `.zarr!` indeed means `.zarr/` directory before we did the naming trick. So anything contains ".zarr!" was a zarr format file.